### PR TITLE
Convert to Qt5

### PR DESCRIPTION
**Pull request by MBach**

### CODE REVIEW

**Comment by randomguy3** [comment]
Thanks for the PR!

Two comments, though: I'd rather not have the style changes (replacing 4 spaces with tabs), and the commit message should say *what* version of Qt you are making it compatible with (I figure this is a Qt4 -> Qt5 conversion, but the commit message should say that, and should preferably indicate what version of Qt you tested it with).

**Comment by MBach** [comment]
Oh sorry about that... I can close this Pull Request and restart with spaces instead of tabs! Qt release was for Qt > 5.x (Latest Ubuntu 15.04 with 5.4.2 I think).

**Comment by randomguy3** [comment]
You should just be able to change your existing commit with `git commit --amend` and force-push it to MBach:master, and that will update the pull request automatically, rather than closing this request and creating a new one.

**Comment by MBach** [comment]
Ok so I think it's better now with commit https://github.com/MBach/mpristester/commit/f1832c64b48d9824cf129a1da846949fdbae3657

File: metadatamodel.h [hunk 1]
```diff
@@ -31,8 +31,8 @@ class MetadataModel : public QAbstractTableModel
     MetadataModel(QObject* parent = 0);
     MetadataModel(const QVariantMap& metadata, QObject* parent = 0);
 
-    int rowCount(const QModelIndex& parent = QModelIndex()) const;
-    int columnCount(const QModelIndex& parent = QModelIndex()) const;
+    int rowCount(const QModelIndex& = QModelIndex()) const;
+    int columnCount(const QModelIndex& = QModelIndex()) const;
```
Comment (randomguy3) [inline metadatamodel.h]: Removing the variable names in *declarations* is unnecessary - the warnings only appear in *definitions*.

**Comment by randomguy3** [comment]
I'd still like to have the commit message be more descriptive. Can you make it something like "Convert to Qt5"?

File: metadatamodel.cpp [hunk 2]
```diff
@@ -86,7 +82,8 @@ void MetadataModel::setMetadata(const QVariantMap& metadata)
 {
     m_metadata = metadata;
     m_keys = m_metadata.keys();
-    reset();
+    ///FIXME
+    ///reset();
```
Comment (randomguy3) [inline metadatamodel.cpp]: beginResetModel() / endResetModel() ?

**Comment by MBach** [comment]
Sure @randomguy3, have a look! I haven't checked everything in details, my goal was just to make it work for my own Music Player https://github.com/MBach/Miam-Player

**Comment by randomguy3** [comment]
Thanks for your work on this!

**Pull request merged**